src/hotkey/hotkey.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cc_option.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+/* TODO(kevyang): add stats for hotkey module */
+
+#define HOTKEY_NSAMPLE   10000 /* keep last 10000 keys sampled by default */
+#define HOTKEY_RATE      100   /* sample one in every 100 keys by default */
+#define HOTKEY_THRESHOLD 10    /* signal for hotkey if 10 or more keys in sample by default */
```

Comment:
Wonder if it makes more sense to use a percentage, so we don't always have to tune HOTKEY_NSAMPLE & HOTKEY_THRESHOLD at the same time. E.g.
```
HOTKEY_THRESHOLD_PCT 0.01
HOTKEY_THRESHOLD (unsigned)(HOTKEY_NSAMPLE * HOTKEY_THREHOLD_PCT)
```